### ISSUE TITLE
Prod issues

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#2055]
- [x] Error `Nothing found at path : contracts/undefined` when clicking on "Go to document". The button should lead to /c/o/organization/orgId/view/members [Yohann]
![Capture d’écran 2020-02-24 à 11.20.20.png](https://images.zenhubusercontent.com/5c6a89ac8de1b31d5443ab99/43722d3a-c09d-4b75-8cce-0fa6c9b19715)
https://sentry.io/organizations/cascade-8/issues/1524796624/?project=1540126&query=user.id%3A%22eLPtspLbkXbr5aewjgrIe4wrFh52%22&statsPeriod=14d

- [ ] Error `Cannot read property 'trim' of null` @ioaNikas  #2121 
on https://archipelcontent.com/catalog/c/o/marketplace/9MxeLYUQrDex7we9xATu/view/main
https://sentry.io/organizations/cascade-8/issues/1527191539/?project=1540126&query=is%3Aunresolved&statsPeriod=14d
Edit: Do you still got this error ?

- [ ] Error `e.created.toDate is not a function` on @Hugo-Hontans #2122 https://archipelcontent.com/catalog/auth/connexion
https://sentry.io/organizations/cascade-8/issues/1525693769/?project=1540126&statsPeriod=14d

- [x] #2062 Error `TypeError: Cannot read property 'orgId' of null` on https://archipelcontent.com/catalog/c/o/dashboard/titles
https://sentry.io/organizations/cascade-8/issues/1524482987/?project=1540126&query=is%3Aunresolved&statsPeriod=14d
This may be related to excel import since I used the `ownerId` column to import the films and attached them to other users.
Linked also to rules blocking users trying to go to /titles on dashboard: https://sentry.io/organizations/cascade-8/issues/1524754872/?project=1540126&query=is%3Aunresolved&statsPeriod=14d

- [x] Error `Cannot read property 'unsubscribe' of undefined` on https://archipelcontent.com/catalog/c/o/marketplace/selection
https://sentry.io/organizations/cascade-8/issues/1526908302/?project=1540126&query=is%3Aunresolved&statsPeriod=14d

- [ ] When you go from /search to a movie page, most of the time you land at half the page, not on top of the page @Maxstgt #2123
Edit: It looks like this issue is a big one and another people encounter it in their project.
https://github.com/angular/angular/issues/24547
TLDR: router property `scrollPositionRestoration` and other workarounds like `window.scrollTo` don't work with dynamic content.
Edit2: Using `html { min-height: 100% }` instead of `height` + `scrollPositionRestoration: top` make it work, but sticky elements are not sticky anymore (toolbar, filters etc)

- [x] Error `Cannot read property 'promotionalElements' of undefined` on [Max] https://archipelcontent.com/catalog/c/o/marketplace/selection
https://sentry.io/organizations/cascade-8/issues/1526904480/?project=1540126&query=is%3Aunresolved&statsPeriod=14d

- [x] Use `jnbHKBP5YLvRQGcyQ8In` as centralOrgId in prod env [Yohann]

- [x] Error: `Cannot read property 'international' of undefined` on [Max] https://archipelcontent.com/catalog/c/o/marketplace/selection
https://sentry.io/organizations/cascade-8/issues/1526905117/?project=1540126&query=is%3Aunresolved&statsPeriod=14d

- [x] Error `Missing or insufficient permissions.` on https://archipelcontent.com/catalog/c/o/marketplace/search
https://sentry.io/organizations/cascade-8/issues/1526859112/?project=1540126&query=is%3Aunresolved&statsPeriod=14d
[Yohann]
Edit: don't know how to reproduce this case. HALP !
> Issue done by Bruce, error on the url: https://archipelcontent.com/catalog/c/o/dashboard/titles/kS84yhjfwQ3SCFGIa3V4/details

**Comment by Jackseed** [comment]
Created dedicated issues for those remaining
